### ISSUE TITLE
Proxyman for iOS is forcibly terminated by jetsam.

### ISSUE THREAD

**Issue by forcemax** [ProxymanApp/Proxyman#924]
### Proxyman version? (Ex. Proxyman 1.4.3)
Proxyman 1.4.0 (10400) for iOS

### macOS Version? (Ex. mac 10.14)
iOS 15.0 beta 2 (iPhone 7), iOS 14.6 (iPhone 2020 SE)

### Steps to reproduce 
I'm trying to log HTTPS requests with proxyman app on iPhone. The proxyman app was set with the following process, and then there was a problem that the recording stopped while testing with the Amazon Shopping app.

1. Enable SSL Proxying in proxyman app and set to record all domains with * in Include.
2. Enable capture traffic.
3. Launch the Amazon Shopping app and enter the product detail page.
4. The VPN icon is disabled in the status bar.

Checking the log with the idevicesyslog command shows that the PacketTunnel process was forcibly terminated by jetsam.

`Jul 7 15:07:58 osanalyticshelper(OSAnalytics)[248] <Notice>: Process PacketTunnel [850] killed by jetsam reason per-process-limit`

### Expected behavior
HTTPS request logging must not have been stopped.

### Screenshots (optional)

Sorry for the bad English.

**Comment by NghiaTranUIT** [comment]
Look like the VPN extension is crashed somewhere when intercepting the Amazon Shopping app.

Just wondering: Are you able to see HTTPS Traffic from other apps/websites? Such as visiting google.com in Safari? 🤔

**Comment by NghiaTranUIT** [comment]
Preliminary investigation shows that "Kill by jetsam" because Proxyman VPN exceeds the memory limitation (https://developer.apple.com/forums/thread/106377)

Look like because you're using iPhone 7. Is there any chance that you can test it on the newest model?

**Comment by forcemax** [comment]
Same thing happened when I tested on iPhone 2020 SE (iOS 14.6).

**Comment by forcemax** [comment]
> Look like the VPN extension is crashed somewhere when intercepting the Amazon Shopping app.
> 
> Just wondering: Are you able to see HTTPS Traffic from other apps/websites? Such as visiting google.com in Safari? 🤔

Yes. I was able to see HTTPS traffic from Safari to google.com using Proxyman for iOS.

However, when testing through the Amazon Shopping app, Proxyman for iOS stopped, so an issue was registered.
